k8s/cse/d2cc/cse-d2cc-gridcapa-adapter-configmap.yaml

```diff
@@ -0,0 +1,38 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cse-d2cc-gridcapa-adapter-configmap
+  labels:
+    app: gridcapa
+    component: gridcapa-adapter
+    region: cse
+    process: d2cc
+data:
+  application.yml: |-
+    spring:
+      rabbitmq:
+        username: ${RABBITMQ_USER}
+        password: ${RABBITMQ_PASSWORD}
+        host: gridcapa-rabbitmq
+        port: 5672
+        template:
+          reply-timeout: 14400000
+      cloud:
+          stream:
+            bindings:
+              handleRun-in-0:
+                binder: rabbit
+                destination: cse-d2cc-launch-task
```

Comment:
Mismatch between this exchange and the one from job-launcher. There are cse-d2cc-run task and cse-d2cc-launch-task. Must be aligned.